Makefile

```diff
@@ -1,7 +1,9 @@
 GOHOSTOS:=$(shell go env GOHOSTOS)
 GOPATH:=$(shell go env GOPATH)
 VERSION=$(shell git describe --tags --always)
-DOCKER ?= podman
+
+DOCKER ?= docker
+DOCKER_CONFIG="${PWD}/.docker"
```

Comment:
what is this config? and where it is used?

Reply:
It's used in `make lint` (which uses a docker image to do the linting). 

We'll probably use it in a few places soon. e.g. we probably want to generate the proto files with a dockerised protoc for consistency.

Reply:
thanks for the clarification mark.